fe/lib/components/QuestionnaireBuilder/FormBuilder/components/NewQuestionForm/components/SelectOptions.tsx

```diff
@@ -99,39 +105,45 @@ export default ({ options, setOptionList }: SelectOptionsProps) => {
 
   return (
     <Stack space="medium">
-      <Stack space="small">
-        <Columns alignY="bottom" space="small">
-          <Column>
-            <Controller
-              control={control}
-              name="option"
-              render={({ field }) => (
-                <TextField
-                  {...field}
-                  id="questionnaireBuilderAddOption"
-                  label="Options"
-                  onClear={() => setValue('option', '')}
-                  ref={optionInputRef}
-                  tone={errors.option ? 'critical' : undefined}
-                />
-              )}
+      {options.length >= RESPONSE_CHOICE_LIMIT ? (
```

Comment:
Should this be `>`

Reply:
That would allow you to add `limit + 1` choices